peachjam/models/judgment.py

```diff
@@ -204,20 +203,3 @@ def get_case_number_string(self):
     def save(self, *args, **kwargs):
         self.string = self.get_case_number_string()
         return super().save(*args, **kwargs)
-
-
-class JudgmentMediaSummaryFile(AttachmentAbstractModel):
```

Comment:
Note to self: none of our databases have data for this model so we don't need a migration.

Reply:
@longhotsummer Does that mean I remove the migrations file?

Reply:
No, the model  migrations are needed, but we don't need to migrate old data (you don't appear to have added that in any case). So It's all good.